includes/class-wc-calypso-bridge-plugins.php

```diff
@@ -108,48 +108,38 @@ public function remove_woocommerce_deactivation_link( $actions, $plugin_file ) {
 	 * Check WooCommerce pages (shop, cart, my-account, checkout) and create them if the following conditions are met.
 	 *
 	 * 1. This is the first time running this method.
-	 * 2. User has not finished Store details task.
-	 * 3. Shop, cart, my-account, and checkout pages do not exist.
+	 * 2. Shop, cart, my-account, and checkout pages do not exist.
 	 *
-	 * @param array $industries Array of industries.
-	 *
-	 * @return array
+	 * @return void
 	 */
-	public function maybe_create_wc_pages( $industries ) {
+	public function maybe_create_wc_pages() {
 		global $wpdb;
 
 		$option_name = 'wc_pages_created_by_wc_calypso_bridge';
 
 		// Abort if we have attempted to create the pages already.
 		if ( 'yes' === get_option( $option_name, 'no' ) ) {
-			return $industries;
+			return;
 		}
 
-		// Abort if the user has completed store details task already.
-		$completed_tasks = get_option( 'woocommerce_task_list_tracked_completed_tasks', [] );
-		if ( in_array( 'store_details', $completed_tasks ) ) {
-			return $industries;
-		}
+		$post_count = (int) $wpdb->get_var( "select count(*) from $wpdb->posts where post_name in ('shop', 'cart', 'my-account', 'checkout')" );
 
-		$post_count = $wpdb->get_var( "select count(*) from $wpdb->posts where post_name in ('shop', 'cart', 'my-account', 'checkout')" );
-
-		// Abort if we find any existing pages.
-		if ( 0 !== (int) $post_count ) {
-			return $industries;
+		// Abort if we don't find all the pages.
+		if ( 4 === $post_count ) {
+			return;
 		}
 
 		// Reset the woocommerce_*_page_id options.
 		// This is needed as woocommerce_*_page_id options have incorrect values on a fresh installation
-		// for an ecom plan. WC_Install:create_pages() might not create all the
+		// for an ecomm plan. WC_Install:create_pages() might not create all the
 		// required pages without resetting these options first.
 		foreach ( [ 'shop', 'cart', 'myaccount', 'checkout' ] as $page ) {
 			delete_option( "woocommerce_{$page}_page_id" );
 		}
 
+		do_action( 'wc_calypso_bridge_maybe_create_pages' );
```

Comment:
Why do we need this? Isn't the `wc_pages_created_by_wc_calypso_bridge` option enough to make a decision?

Reply:
I wanted to log `time()` and `wc_pages_created_by_wc_calypso_bridge` gives a yes/no .
I also wanted to keep it consistent with the rest of the loggers

Reply:
OK, removing it, pushing and merging! Thanks!